### DISCUSSION TITLE
Instructions for llm-cli obsolete?

### DISCUSSION THREAD

**Discussion by MSSRPRAD** [rustformers/llama-rs#404] [General]
I get this error when I do:
```
llm gptneox infer --model-path '/home/pradyumnamalladi/Downloads/RedPajama-INCITE-Base-3B-v1-q4_0.bin' -p "Hello!";
⣯ Loaded hyperparametersError: 
   0: Could not load model
   1: non-specific I/O error
   2: failed to fill whole buffer

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Pls guide on how to run these models now:

```
RedPajama-INCITE-Base-3B-v1-q4_0.bin
llama-2-7b-chat.ggmlv3.q2_K.bin
llama-7b.ggmlv3.q2_K.bin
```

**Comment by philpax** [comment]
Hi there, thanks for reporting this! Yes, the released CLI is quite out of date now; I've updated the README to indicate that you should [install from source](https://github.com/rustformers/llm#installing-from-source) instead. Does that solve the problem for you?

**Reply by MSSRPRAD** [reply]
Yeah it worked great when I cloned the repo and ran it so it's fine.

Does this run as fast as llama.cpp btw?
